src/main/scala/nl.knaw.dans.easy.properties/app/repository/sql/SQLStateDao.scala

```diff
@@ -53,36 +55,27 @@ class SQLStateDao(implicit connection: Connection, errorHandler: SQLErrorHandler
   override def getById(ids: Seq[String]): QueryErrorOr[Seq[State]] = {
     trace(ids)
 
-    executeGetById(parseState)(QueryGenerator.getElementsById("State", "stateId"))(ids)
+    executeGetById(parseState)(QueryGenerator.getElementsById(daoName, "stateId"))(ids)
   }
 
   override def getCurrent(ids: Seq[DepositId]): QueryErrorOr[Seq[(DepositId, State)]] = {
     trace(ids)
 
-    executeGetCurrent(parseDepositIdAndState)(QueryGenerator.getCurrentElementByDepositId("State"))(ids)
+    executeGetCurrent(parseDepositIdAndState)(QueryGenerator.getCurrentElementByDepositId(daoName))(ids)
```

Comment:
Using `daoName` for `tableName`, here and in many other situations. In many dao's the value is same but this might lead to confusion.